content/foundations/glossary.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Glossary
+---
+
+#### Brand theme
+
+Includes color scales, functional colors, spacing, fonts, etc.
+
+#### Color mode
+
+The appearance mode set by the operating system (light, dark, auto).
+
+#### Scheme
+
+Name for the "theme" in the UI appears to user in Appearance settings.
```

Comment:
The way this sentence is worded is a little confusing to me. 

I think included scheme name examples could help clarify (e.g. `light, dark, dark dimmed`)

Reply:
Yeah this sentence is a bit confusing 😬